src/modules/fancyzones/dll/dllmain.cpp

```diff
@@ -156,9 +157,19 @@ class FancyZonesModule : public PowertoyModuleIface
     {
         app_name = GET_RESOURCE_STRING(IDS_FANCYZONES);
         app_key = NonLocalizable::FancyZonesStr;
-        std::filesystem::path logFilePath(PTSettingsHelper::get_module_save_folder_location(app_key));
+        const auto appFolder = PTSettingsHelper::get_module_save_folder_location(app_key);
+        const std::filesystem::path logFolder = LoggerHelpers::get_log_folder_path(appFolder);
+        
+        std::filesystem::path logFilePath(logFolder);
         logFilePath.append(LogSettings::fancyZonesLogPath);
         Logger::init(LogSettings::fancyZonesLoggerName, logFilePath.wstring(), PTSettingsHelper::get_log_settings_file_location());
+        
+        std::filesystem::path oldLogFolder(appFolder);
+        oldLogFolder.append(LogSettings::fancyZonesOldLogPath);
+        LoggerHelpers::delete_old_log_folder(oldLogFolder);

```

Comment:
I think it's better to add removing old logs(`FancyZonesLogs`) to the installer.
Something as was made in https://github.com/microsoft/PowerToys/pull/10438.

Reply:
That change had a different purpose and it was only a temporarily change to the bootstrapper just of 0.35, it will be removed in 0.37.
We shouldn't clean up logs in the bootstrapper because for enterprise users, the bootstrapper is not used (they use the msi).

Reply:
This part should also be removed eventually.